braindecode/models/hybrid.py

```diff
@@ -109,7 +110,7 @@ def forward(self, x):
             )
 
         merged_out = torch.cat((deep_out, shallow_out), dim=1)
-        linear_out = self.final_conv(merged_out)
+        linear_out = self.final_layer(merged_out)
         softmaxed = nn.LogSoftmax(dim=1)(linear_out)
         squeezed = softmaxed.squeeze(3)
```

Comment:
These should be included in `final_layer`. Squeezing may be replaced with `Expression`?

Reply:
Done. I left the squeeze there, if you all decide to just replace with expression let me know